src/components/material.js

```diff
@@ -121,6 +122,7 @@ module.exports.Component = registerComponent('material', {
     material.transparent = data.transparent !== false || data.opacity < 1.0;
     material.vertexColors = parseVertexColors(data.vertexColors);
     material.visible = data.visible;
+    material.blending = parseBlending(data.blending);
 
     // Check if material needs update.
     if (Object.keys(oldData).length &&
```

Comment:
If you want another easy task, we should remove this Object.keys call to save an array creation. Instead use a for...in loop to check for key existence or something.

Reply:
That works. You don't even need the `if (key)` check I believe.